packages/airnode-deployer/terraform/airnode/gcp/modules/function/main.tf

```diff
@@ -85,7 +77,16 @@ resource "google_cloudfunctions_function" "function" {
   timeout               = var.timeout
   max_instances         = var.max_instances
   environment_variables = merge(merge(var.environment_variables, fileexists(var.secrets_file) ? jsondecode(file(var.secrets_file)) : {}), { AIRNODE_CLOUD_PROVIDER = "gcp" })
+}
 
+resource "random_string" "scheduler_service_account_id" {
```

Comment:
Why is it called `random_string`?

Reply:
I guess because it's a string that's random :smile: It's a Terraform resource name, that's not my naming.